references/kvm-admin-api/apiproxy/kvm-admin-v1.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!--
+ Copyright 2021 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+
+<APIProxy name="kvm-admin-v1">
```

Comment:
so that this file doesn't need to be kept up to date, please can you make it similar to  https://github.com/apigee/devrel/blob/main/references/cicd-jenkins/airports-cicd-v1/apiproxy/airports-cicd-v1.xml so it is just 4 lines long?